modules/FvwmMFL/FvwmMFL.c

```diff
@@ -849,6 +766,31 @@ set_socket_pathname(void)
 
 	xasprintf(&socket_name, "%s/fvwm_mfl_%s.sock", socket_basepath,
 		display_env);
+
```

Comment:
What about naming the file `fvwm_mfl_%s.lock` instead?  But that's purely cosmetical...

Reply:
This also puts all the sockets (if fvwm is running on multiple DISPLAY's at the same time) in a single directory, so a bit for organization too. But as you said mostly cosmetical, I'm just stating the logic of this choice.

Reply:
With Thomas' proposal, you would have the files in `ls` output grouped by type, with my proposal grouped by display.  Not sure what is more meaningful.

Reply:
Oh, to clarify: I mean using `"%s/fvwm_mfl_%s.lock"` (with the leading `%s/` of course) instead of using `"%s/fvwmmfl_lockfile_%s.lock"`.

Reply:
Thanks for the clarification, though this is the location of the socket file, so I would use `.sock` and not `.lock`.

Reply:
It's not changing.